src/AirshipBindings.Android.Core/Transforms/Metadata.xml

```diff
@@ -25,7 +25,8 @@
   <remove-node path="/api/package[@name='com.urbanairship.job']/class[@name='Job']" />
   <remove-node path="/api/package[@name='com.urbanairship.job']/class[@name='JobDispatcher']" />
   <remove-node path="/api/package[@name='com.urbanairship.job']/class[@name='AndroidJobScheduler']" />
-	
+  <remove-node path="/api/package[@name='com.urbanairship.channel']/class[@name='NamedUser']/method[@name='addNamedUserListener']" />
```

Comment:
The bindings generator turns listener adders into C# events, and the generated event was called NamedUser which conflicted with the class. Removing it was the easiest solution, since it's for internal use and we don't need to expose it anyway.